server/work-queue/jobs/send-email.js

```diff
@@ -0,0 +1,48 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict';
+
+var _ = require('underscore');
+var sendmail = require('sendmail')();
+
+var log = require('../../logger')('server.work-queue.jobs.send-email');
+var config = require('../../config');
+
+var contentTmpl = _.template('Welcome to Chronicle, <%= email %>!');
+
+module.exports = {
+  // o is an object with keys { email }
+  perform: function(o, cb) {
+    var toEmail = o.email;
+    log.verbose('job created with params ' + JSON.stringify(o));
+
+    switch (config.get('env')) {
+      case 'local':
+        sendmail({
+          from: 'info@chronicle.firefox.com',
+          to: toEmail,
+          subject: 'Welcome to Chronicle',
+          content: contentTmpl({email: toEmail})
+        }, function (err, reply) {
+          if (err) {
+            log.error('Unable to send email: ' + err);
+            cb(err);
+            throw err;
```

Comment:
I played with the queue a bit more yesterday; I think `cb(err)` tells node-resque the job failed, and we don't need to throw. Sorry I've been unclear on this point.